dev/bots/analyze-sample-code.dart

```diff
@@ -212,50 +219,40 @@ dependencies:
       <String>['analyze', '--no-preamble', '--no-congratulate', mainDart.parent.path],
       workingDirectory: temp.path,
     );
-    stderr.addStream(process.stderr);
-    final List<String> errors = await process.stdout.transform<String>(utf8.decoder).transform<String>(const LineSplitter()).toList();
-    if (errors.first == 'Building flutter tool...')
+    final List<String> errors = <String>[];
+    errors.addAll(await process.stderr.transform<String>(utf8.decoder).transform<String>(const LineSplitter()).toList());
+    errors.addAll(await process.stdout.transform<String>(utf8.decoder).transform<String>(const LineSplitter()).toList());
+    if (errors.isNotEmpty && (errors.first.contains(' issues found. (ran in ') || errors.last.contains(' issue found. (ran in '))) {
+      errors.removeAt(0); // the "23 issues found" message goes onto stderr, which is concatenated first
```

Comment:
But you use `errors.last` in the cast of a single issue found?

Reply:
oops

Reply:
fixed